services/ui-src/src/utils/getLabelText.tsx

```diff
@@ -11,6 +11,7 @@ export interface LabelData {
   label: string;
   text: string;
   id: string;
+  excludeFromIds?: string[]; //currently used to hide qualifiers from some categories
```

Comment:
Suggestion: This would be a great spot for a JSDoc comment, so that it will be available on mouseover anywhere in the codebase.

```suggestion
  /**
   * We use this to hide certain qualifiers from certain categories.
   *
   * For example, in AIS-AD, we don't ask about the Zoster vaccine
   * for the 19-65 age group.
   */
  excludeFromIds?: string[];
```